apps/react/src/features/user/components/AvatarUpload/AvatarUpload.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useRef, useState } from 'react';
+import { Alert, Avatar, Box, Button } from '@mui/material';
+
+import { typedMemo } from '@js-camp/react/utils/typedMemo';
+
+import styles from './AvatarUpload.module.css';
+
+interface Props {
+
+	/** Avatar image url. */
+	readonly imageUrl: string | null;
+
+	/** Change avatar image. */
+	readonly changeImage: (image: File | null) => void;
+
+	/** Is disable. */
+	readonly disable?: boolean;
+
+	/** Class name. */
+	readonly className?: string;
+}
+
+/** Max image file size - 60 Kb. */
+const MAX_IMAGE_FILE_SIZE = 1024 * 60;
+
+/** Accept image file types. */
+const ACCEPT_FILE_TYPES = ['image/jpeg', 'image/png', 'image/webp'];
+
+/** Error display time. */
+const ERROR_DISPLAY_TIME = 1000 * 3;
+
+/**
+ * Avatar upload component.
+ * @param props Props.
+ */
+const AvatarUploadComponent: FC<Props> = props => {
+	const [error, setError] = useState<string | null>(null);
+	const errorTimer = useRef<NodeJS.Timeout | null>(null);
```

Comment:
We are working in browser environment here, not in `NodeJS` :)
```suggestion
	const errorTimer = useRef<number | null>(null);
```

Reply:
how then to pass `setTimeout` to `errorTimer?`